### ISSUE TITLE
how can use postcss in Global css

### ISSUE THREAD

**Issue by didiself** [nuxt/nuxt.js#846]
nuxt.config.js
```
css: [
    {src: 'normalize.css'},
    {src: '~assets/sass/common.scss', lang: 'scss'}
  ],
  build: {
    postcss: [
      require('autoprefixer')({
        browsers: ['last 2 versions', 'Firefox ESR', '> 1%', 'ie >= 8', 'iOS >= 8', 'Android >= 4']
      }),
      require('postcss-pxtorem')({
        rootValue: 100,
      })
    ],
}
```

common.scss
```
body{
  display: flex;
  transition:all 1s;
  a{
    color:#ff0
  }
}
```

output:
```
body {
    display: flex;
    transition: all 1s;
}
```

so, how can I use autoprefixer in global scss?

also, I try to use loaders , but ERROR.

nuxt.config.js:
```
{
            test: /\.scss$/,
            use: [
              'vue-style-loader',
              'style-loader',
              'css-loader',
              {
                loader: 'postcss-loader',
                options: {
                  plugins: () => [
                  require('postcss-pxtorem')({
                    rootValue: 100,
                  }),
                  require('autoprefixer')({
                    browsers: ['last 2 versions', 'Firefox ESR', '> 1%', 'ie >= 8', 'iOS >= 8', 'Android >= 4']
                  }),
                ]
              },
          },
          'sass-loader'],
      }
```

![image](https://cloud.githubusercontent.com/assets/14201030/26821503/3745e2a2-4ad9-11e7-8bd4-3202f55ffc5f.png)

**Comment by didiself** [comment]
nuxt.config.js

![image](https://cloud.githubusercontent.com/assets/14201030/26821694/dca0b6d2-4ad9-11e7-9295-aac4c8e6f5d8.png)

**Comment by Atinux** [comment]
Hi @didiself 

Here the solution: https://glitch.com/edit/#!/nuxt-postcss-global?path=nuxt.config.js:1:0
Demo: https://nuxt-postcss-global.glitch.me/

I am currently working to add postcss loader directly into nuxt.js core for global css files.

**Comment by didiself** [comment]
thanks a lot, it's work , @Atinux

**Comment by doriandrn** [comment]
Thanks for this @Atinux! Additionally, it would be great if we could have the postcss config readen from postcss.config.js file instead of just registering the plugins in build: {}.
https://github.com/postcss/postcss-loader

**Comment by irrg** [comment]
Am I crazy for thinking this worked previously? I've been developing with the useless version 'latest' in my package.json so when I switched it to the latest alpha today (no clue what version I was working against, noob mistake) my global css postcss stopped working.

**Comment by doriandrn** [comment]
@irrg Had the same issue. Made it work like this:

```js
let cssLoader = config.module.rules.find((loader) => loader.test.toString() === '/\\.css$/')
cssLoader.use.splice(2, 0, {
  loader: 'postcss-loader',
  options: {...}
})
```

Please note you'll have to move your postcss configuration to `postcss.config.js` in your root folder. More info about this here: https://github.com/postcss/postcss-loader

**Comment by irrg** [comment]
@doriandrn I got it to work, but thanks for those pointers either way. I'm mostly trying to figure out how to keep track of BC-breaking changes, which this seems to be. When'd this behavior change?

**Comment by alancwoo** [comment]
Sorry to post in a closed issue, but I am on 1.0.0-rc11 and can't even seem to get basic global css to work whatsoever. I've created a [stackoverflow issue](https://stackoverflow.com/questions/48039118/nuxt-js-vue-with-global-postcss) in case it is not exactly relevant to this issue in particular, but none of the suggestions here are working for me
